leapp/cli/upgrade/__init__.py

```diff
@@ -230,6 +231,7 @@ def upgrade(args):
         workflow.run(context=context, skip_phases_until=skip_phases_until, skip_dialogs=True)
 
     report_errors(workflow.errors)
+    report_inhibitors(context)
```

Comment:
`report_info` below uses `workflow.failure` but that is only set to `True` elsewhere when an error is raised. If all errors get filtered out as inhibitors, `report_info` spits out green blocks as if the upgrade succeeded and the exit code is 0 instead of 1.
```suggestion
    fail = workflow.failure or report_inhibitors(context)
```
For this we'd also need `report_inhibitors` to return a bool - see below.

Also line 238 (GitHub doesn't allow me to suggest reviews that far):
`report_info(report_files, log_files, fail=fail)`

(Or maybe you know a way to set `workflow.failure` when inhibitors appear. I couldn't figure that out.)